lib/reunion.rb

```diff
@@ -0,0 +1,43 @@
+class Reunion
+
+  attr_reader :name,
+              :activities
+
+  def initialize(name)
+    @name = name
+    @activities = []
+  end
+
+  def add_activity(activity)
+    @activities << activity
+  end
+
+  def total_cost
+    @activities.map do |act|
+        act.total_cost
+    end.flatten.sum
+  end
+  #
+  def breakout
```

Comment:
A couple of things are tripping you up here:
1. You are resetting the value of the `debt_amt` variable to the return of the `each` block on line 25 (remember what `each` returns!)
2. You are calling the variable `debt_amt` inside the outer `each` - you probably meant to call this at the end of the method instead.